srml/balances/src/lib.rs

```diff
@@ -108,15 +226,15 @@ pub struct BalanceLock<Balance, BlockNumber> {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Balances {
-		/// The total amount of stake on the system.
+		/// The total amount of stake in the system.
```

Comment:
Can we change this to say "Total amount of units issues in the system"

The word "stake" is an artifact of this being called "TotalStake" in the past, but may be confusing in context to the staking module